app/models/post.rb

```diff
@@ -5,14 +5,18 @@ class Post < ApplicationRecord
 
   after_create :update_posts_counter
 
-  validates :title, presence: true, length: { maximum: 250 }
-  validates :comments_counter, numericality: { greater_than_or_equal_to: 0 }
-  validates :likes_counter, numericality: { greater_than_or_equal_to: 0 }
+  # validates :title, presence: true, length: { maximum: 250 }
+  # validates :comments_counter, numericality: { greater_than_or_equal_to: 0 }
+  # validates :likes_counter, numericality: { greater_than_or_equal_to: 0 }
 
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
well resolved! ✔️